docker-compose.yml

```diff
@@ -7,15 +7,16 @@ services:
       dockerfile: Dockerfile.builder
       args:
         grobid_tag: ${GROBID_TAG}
-    image: elifesciences/sciencebeam-trainer-grobid-builder:${GROBID_TAG}-${IMAGE_TAG}
+    image: elifesciences/sciencebeam-trainer-grobid-builder:${IMAGE_TAG}
   sciencebeam-trainer-grobid:
     build:
       context: .
       dockerfile: Dockerfile
       args:
-        builder_image: elifesciences/sciencebeam-trainer-grobid-builder:${GROBID_TAG}-${IMAGE_TAG}
+        builder_image: elifesciences/sciencebeam-trainer-grobid-builder:${IMAGE_TAG}
         grobid_tag: ${GROBID_TAG}
-    image: elifesciences/sciencebeam-trainer-grobid:${GROBID_TAG}-${IMAGE_TAG}
+        image_tag: ${IMAGE_TAG}
```

Comment:
naming the arguments `*_tag` is perhaps consistent with other project. I'm starting to wonder whether they should be named differently because tag is something that is put on the image after it is built, and there may be multiple ones like `1.2.3-12345678` vs `1.2.3`. These can be `grobid_version` (explicitly versioned) and `revision` (just a commit sha).

Reply:
I revised it to be just the commit